lib/merit/rule.rb

```diff
@@ -7,19 +7,14 @@ class Rule
                   :multiple, :temporary, :score, :block, :category
 
     # Does this rule's condition block apply?
-    def applies?(target_obj = nil)
+    def applies?(target_obj = nil, current_user = nil)
```

Comment:
Both of these ` = nil` seem like a bug to me. I think these arguments should always be required.